test/README.md

```diff
@@ -434,3 +434,14 @@ to host the images.
 
 Runs on the hypervisor. Responsible for launching all of the VMs that
 are used in the test step.
+
+Adding the env variable `PERIODICS` it will create the scenarios defined
```

Comment:
Can we make it a longer name? There's too much change of collision this way, I think

Reply:
Name is changed now and more "non-CI specific".